app/static/js/new_post.js

```diff
@@ -64,7 +64,8 @@ $(function () {
     // initialize tinymce editor
     tinymce.init({
```

Comment:
Small issue. The underline option isn't shown in the toolbar but the keyboard shortcut works.

Reply:
Okay will add in toolbar option.